framework/encode/custom_dx12_wrapper_commands.h

```diff
@@ -248,6 +248,8 @@ struct CustomWrapperPostCall<format::ApiCallId::ApiCall_ID3D12Device4_CreateComm
     }
 };
 
+// TODO: add CustomWrapperPostCall struct for ID3D12Device4::CreateReservedResource1
```

Comment:
Capture support for `ID3D12Device4::CreateReservedResource1` was added in commit: https://github.com/LunarG/gfxreconstruct/commit/5aa149eae301191c40e4dad468d76739e408e71c

Could you rebase onto the latest in `dev` and update this PR as needed?

Reply:
Done.